databases/week3/meal_sharing_query.sql

```diff
@@ -0,0 +1,162 @@
+USE meal_schema;
+
+-- Get all meals
+
+SELECT * 
+FROM Meal;
+ 
+-- Add a new meal
+INSERT INTO meal
+VALUE(1,'Doro wot', 'chicken stew', 'Tagensvej 43','2020-08-22', 5, (175*5), '2020-08-19'),
```

Comment:
The meal table will insert the primary key id automatically. The `id` is not needed to be defined.

Reply:
I get this error when executing your query
`Error Code: 1136. Column count doesn't match value count at row 1`
It means the number of fields do not match with the table's

Reply:
its fixed now i tested all the queries and it worked

Reply:
i pushed the change you can look into it